src/PRExample.ts

```diff
@@ -0,0 +1,39 @@
+import { data, students, teachers } from "./data";
+import { callbackify } from "util";
+
+const allstudents = () => {
```

Comment:
Use camelCase

Reply:
Fixed the camel case issue. Changed the function name from allstudents to allStudents